### PR TITLE
11 build prof bodies open source title huge events

### PR DESCRIPTION
**Pull request by riyavishwa981**
Created Professional Bodies section
Created TcetOpenSource Section
Created Events section

### CODE REVIEW

**Review by demonkillerr — COMMENTED** [review]
code looks good to me

File: src/components/ProfessionalBodies.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            </div>
+            <div className="text-slate-700 font-Inter text-lg">
+              Working together for a better Future!!
+            </div>
+          </div>
+          <div className="md:flex-col sm:w-3/5">
+            {/* Shastra */}
+            <div className="flex justify-center items-center bg-gray-200 rounded-lg p-6">
+          
+                <img
```
Comment (JayeshVP24) [inline src/components/ProfessionalBodies.jsx]: use `<Image>` tag of @astro/image/components

File: public/imgs/tcet_shastra.png
Comment (JayeshVP24) [inline public/imgs/tcet_shastra.png:1]: rename images folder to logos

File: public/imgs/pic_1.png
Comment (JayeshVP24) [inline public/imgs/pic_1.png:1]: move to events folder

File: public/imgs/astronaut.png
Comment (JayeshVP24) [inline public/imgs/astronaut.png:1]: move to folder misc

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

**Review by JayeshVP24 — APPROVED** [review]

**Pull request merged**